templates/analysis/analysis_form.html

```diff
@@ -1,21 +1,48 @@
 {% extends "generic/object_form.html" %}
+{% load form_tags %}
+{% block title_bar_end %}
+  {% if analysis.id %}
+    <div class="join">
+      <div class="tooltip" data-tip="Configure">
+        <a class="btn btn-primary join-item"
+           href="{% url 'analysis:configure' team.slug analysis.id %}">
+          <i class="fa-solid fa-gear"></i>
+        </a>
+      </div>
+      <div class="tooltip" data-tip="Delete">
+        <a class="btn btn-primary join-item rounded-r-full"
+           href="{% url 'analysis:delete' team.slug analysis.id %}">
+          <i class="fa-solid fa-trash"></i>
+        </a>
+      </div>
+    </div>
+  {% endif %}
+{% endblock title_bar_end %}
+{% block pre_form %}
+  {% if analysis.needs_configuration %}
+    <div class="alert mt-2">
+      <i class="fa-solid fa-circle-info"></i>
+      <span>This analysis requires static <a class="link" href="{% url "analysis:configure" request.team.slug analysis.id %}">configuration</a></span>
```

Comment:
Just wondering, would it be obvious to the user what "static config" refers to?

Reply:
Maybe not but it's what I came up with - happy to change it if you have suggestions. Making this more of a wizard with some help text would improve things too.